scripts/plugins/deploy_plugins.py

```diff
@@ -51,10 +51,20 @@ def Deploy_Plugins():
     os.environ["BITOPS_DEFAULT_ROOT_DIR"] = BITOPS_default_folder
 
     # Global environment evaluation
+    # TODO: Drop support for 'ENVIRONMENT' env var
+    if 'ENVIRONMENT' in os.environ:
+        logger.warning(
+            "'ENVIRONMENT' var is deprecated in v2.0.0 and will be removed in the future versions! "
+            "Use the 'BITOPS_ENVIRONMENT' env var instead!"
```

Comment:
Now that `ENVIRONMENT` is deprecated in `v2.0.0` in favor of new `BITOPS_ENVIRONMENT`, but we're handling the case gracefully.

The question though, at what point do we want to drop the `ENVIRONMENT` var completely?

Reply:
i'd create an issue/discussion so we can discuss (also to add it to the roadmap)

Reply:
Created a note here: https://github.com/bitovi/bitops/discussions/245#discussioncomment-3399371 in the "Roadmap" Discussion so it's not lost.